File: 136/136.cpp
/*!***********************
* @brief:  leetcode-136-只出现一次的数字
* @date:   2019/5/7
* @envir:  gcc 7.4.0
* @author: luhao
* ***********************/

class Solution {
public:
    int singleNumber(vector<int>& nums) {
        int ans = 0;
        for (int i=0; i<nums.size(); i++){
                ans ^= nums[i];
        }
        return ans;
    }
};

File: 136/136.py
# @brief:  leetcode-136-只出现一次的数字
# @envir:  python 2.7
# @date:   2019/5/7
# @author: luhao

class Solution(object):
    def singleNumber(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        ans = 0
        for i in nums:
	        ans ^= i
        return ans
        
